src/common.c

```diff
@@ -201,16 +202,29 @@ const char* GetShardUniqueId() {
         RedisModule_Assert(RedisModule_CallReplyType(reply) == REDISMODULE_REPLY_ARRAY);
         RedisModuleCallReply *uuidReply = RedisModule_CallReplyArrayElement(reply, 1);
         RedisModule_Assert(RedisModule_CallReplyType(uuidReply) == REDISMODULE_REPLY_STRING);
+        
         size_t len;
         const char* logFileName = RedisModule_CallReplyStringPtr(uuidReply, &len);
-        const char* last = strrchr(logFileName, '/');
-        if(last){
-            len = len - (last - logFileName + 1);
-            logFileName = last + 1;
+        char uuid[64];
+        if (*logFileName == '\0' || *logFileName == '\n' || *logFileName == '\r' || *logFileName == ' ') {
+            RedisModule_Log(ctx, "notice", "No log file is available");
+            uuid_t binuuid;
+            uuid_generate_random(binuuid);
```

Comment:
@rafie uuid is problematic because it will be different each time you start the redis